x-pack/plugins/security_solution/public/detections/components/rules/eql_query_bar/eql_overview_link.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { EuiLink, EuiText } from '@elastic/eui';
+import { EQL_OVERVIEW_LINK_TEXT } from './translations';
+
+const EQL_OVERVIEW_URL = 'https://eql.readthedocs.io/en/latest/query-guide/index.html';
```

Comment:
Chatting with folks it looks like we want to link out to the elastic docs:

https://www.elastic.co/guide/en/elasticsearch/reference/master/eql.html